### DISCUSSION TITLE
MCOHome MH3901-Z

### DISCUSSION THREAD

**Discussion by pjator** [zwave-js/zwave-js#3486] [Q&A]
Hello I did not want to make a pull request since I assume adding a device is not a pull request but rather something I should try to contribute to. But to be honest my competence is to low in this area. But I have bought a MCOHome MH3901-Z which is an exellent Z-wave based thermostat for people with under floor heating which is controlled via 24V valves. 

It works well in my home assistant but there are some settings that can only be reached from the Z-wave parameter settings and since the device is not in the database it is not possible to make the changes to the device. I have uploaded the usermanual from MCOHome with the parameter sheet. Could anyone be kind enought to look at adding this device to the Z-wave JS config DB. 
Best regards
Peter
[mh3901-z.pdf](https://github.com/zwave-js/node-zwave-js/files/7317052/mh3901-z.pdf)
.

**Comment by AlCalzone** [comment]
We have an [issue template](https://github.com/zwave-js/node-zwave-js/issues/new?assignees=&labels=config+%E2%9A%99&template=device_config.yml&title=Missing+device+configuration%3A+%3Center+manufacturer+and+device+name%3E) for requesting new device files - please fill it out.
However, you're likely to get support quicker if you try to provide a config file yourself. Our documentation has quite an extensive section on this.

Due to the sheer amount of devices, we don't have the capacity to write every file ourselves.
